plugin/manager/manager.go

```diff
@@ -0,0 +1,240 @@
+package manager
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+
+	"github.com/notaryproject/notation-go/plugin"
+)
+
+const (
+	// NamePrefix is the prefix required on all plugin binary names.
+	NamePrefix = "notation-"
```

Comment:
Nit, rename to `PluginPrefix` ?

Reply:
I've found that this variable is duplicated in `plugin` and `manager`. I'll remove the later and rename the former to just `Prefix`, so users will use it as `plugin.Prefix`